kubernetes/annotations_and_labels.md

```diff
@@ -19,14 +19,23 @@
 ## Common Labels (for guest cluster services)
 
 - `app` - value should contain the name of the application. Should be applied
-  to every Kubernetes resource associated with an application. This should 
+  to every Kubernetes resource associated with an application. This should
   also be used as replicas selector. E.g. `app=nginx-ingress-controller`.
   Exceptions can be made to accomodate for adherence to existing selectors
   upstream.
 - `giantswarm.io/service-type` - value should be either `system` for core system
   services (i.e. K8s components) or `managed` for Giant Swarm managed services
   (i.e. networking, DNS, monitoring, ingress controller, etc.). Latter would be
   managed by `chart-operator`.
+- `release.giantswarm.io/version` - value should be the used Giant Swarm release
+  version, e.g. `2.3.0`. Should be applied to all Kubernetes nodes, regardless
+  their role.
+- `operator.giantswarm.io/version` - value should be the used operator version
+  bundle version, e.g. `1.0.0`. Should be applied to all Kubernetes nodes,
+  regardless their role.
+- `operator.giantswarm.io/provider` - value should be the installation's
```

Comment:
The version labels LGTM but I think this could just be `giantswarm.io/provider` as to me this isn't operator specific.

Reply:
Done.